src/components/app-root/app-root.tsx

```diff
@@ -41,6 +41,7 @@ export class AppRoot {
               <stencil-route url="/guidelines" component="page-guidelines" />
               <stencil-route url="/activity" component="page-activity" />
               <stencil-route url="/privacy" component="page-privacy" />
+              <stencil-route component="page-home" />
```

Comment:
@mojowen I don't think this is valid. It looks like this successfully deployed to `next`; however, if you visit an invalid route you get this:
![error](https://user-images.githubusercontent.com/2229946/97051230-d0389900-154c-11eb-99fe-6701270c5c5c.png)